TypeError: analyze_friends() takes exactly 6 arguments (5 given)

Running the app locally with Python 2.7.14 (as root)
After the "Enter your pincode?" prompt, I see:

Generating and signing request for an access token...

          	 nonbinary	       men	     women	   unknown
Traceback (most recent call last):
  File "analyze.py", line 356, in <module>
    tok, tok_secret)),
TypeError: analyze_friends() takes exactly 6 arguments (5 given)

I ran in debug mode (-vv) and didn't see anything interesting before the program dumped. Thoughts?

the unit test fails, thusly:

# python2.7 -m unittest discover -v
test_declared_gender (tests.test_declared_gender.TestDeclaredGender) ... FAIL

======================================================================
FAIL: test_declared_gender (tests.test_declared_gender.TestDeclaredGender)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/twitter-gender-distribution/tests/test_declared_gender.py", line 50, in test_declared_gender
    description, expected_gender, guess))
AssertionError: Should have guessed profile 'mum to one boy and one girl' was 'female', not 'male'

----------------------------------------------------------------------
Ran 1 test in 0.115s

FAILED (failures=1)

--- a/analyze.py
+++ b/analyze.py
@@ -352,7 +352,7 @@ if __name__ == '__main__':
         '', 'nonbinary', 'men', 'women', 'unknown'))
 
     for user_type, users in [
-        ('friends', analyze_friends(user_id, consumer_key, consumer_secret,
+        ('friends', analyze_friends(user_id, None, consumer_key, consumer_secret,
                                     tok, tok_secret)),
         ('followers', analyze_followers(user_id, consumer_key, consumer_secret,
                                         tok, tok_secret)),

